apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -433,6 +433,10 @@ Please choose a different model name.
 		StorageProviderRegistry: storageProviderRegistry,
 	})
 	if err != nil {
+		// Clean up the environ.
+		if e := broker.Destroy(m.callContext); e != nil {
+			logger.Warningf("failed to destroy environ, error %v", e)
```

Comment:
We're using "model" not "environ", so let's tweak the wording, here and in the comment.
Maybe tweak the IAAS code as a drive by as well.